src/loader/loader.c

```diff
@@ -7,15 +7,15 @@
 
 #include <access/xact.h>
 #include <access/heapam.h>
-#include "../compat-msvc-enter.h"
+#include "../compat/compat-msvc-enter.h"
```

Comment:
Isn't the include path set so that we do not need to reference the parent directory? This seems a little fragile.

Reply:
it is not